### PR TITLE
Forrce -> PokemonDeadChannel

### PR DESCRIPTION
**Pull request by ghost**
something something i know about dragon but just for consistency's sake

### CODE REVIEW

**Comment by HoeenCoder** [comment]
wait

**Comment by ghost** [comment]
Take off the DONT MERGE tag considering what has been revealed earlier

**Comment by Torracat** [comment]
Is there a way to ignore a travis error? the irregular white space is necessary

File: data/mods/ssb/statuses.js [hunk 1]
```diff
@@ -1006,16 +978,28 @@ let BattleStatuses = {
...
+			this.add(`c|+PokemonDeadChannel|づ｀･ω･)づ∴“`);
+			this.add(`c|+PokemonDeadChannel|づづづづづ∴≒`);
+			this.add(`c|+PokemonDeadChannel|づづづづづづ∵`);
+			this.add(`c|+PokemonDeadChannel|(_／⌒ヽ_)∴`);
+			this.add(`c|+PokemonDeadChannel|ORA ORA ORA ORA ORA !!!`);
 		},
 		onSwitchOut() {
-			this.add(`c|&pre|sometimes science is more art than science`);
+			this.add(`c|+PokemonDeadChannel|Bye Losers`);
+			this.add(`c|+PokemonDeadChannel|　＜⌒／ヽ-､_＿_`);
```
Comment (ghost) [inline data/mods/ssb/statuses.js]: would it possible to ignore this line specifically in the travis ci test? The irregular white space is necessary

Comment (HoeenCoder) [reply data/mods/ssb/statuses.js]: sure

Comment (ghost) [reply data/mods/ssb/statuses.js]: i see. enlighten me

**Pull request merged**